docs/direction/2026-08-26-ray-directives.md

```
@@ -0,0 +1,182 @@
+>   - provide every command run on how and what was searched
+>
+> 7. are we properly using the existing deep extraction and reflection and generated artifacts to help in this specific task and research?
+>
+> 8. how can we use the pre-existing graphify native deep extraction to understand what are missing gaps in our design or to provide an honest assessment if this project's goal of using graphify as a knowledge base for research and understanding its dependencies or aid in research or conserve context and tokens
+>    - are the communities/nodes/edges being properly connected to aid in this project?
+>
+> 9. are there AST tree-sitter/LSP tools/or other tools we can use to help analyze our config files better?
+>
+> 10. can we generate and keep in sync the following:
+>    - tabular report of modular skill(s) -> mise task(s) - python library module(s)/function(s) -> graphify sdk/cli
+>    - workflow/sequence diagrams of the above
```

markdownlint-cli2 (0.23.2)

[warning] Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)